src/test/java/htsjdk/samtools/cram/build/ContainerFactoryTest.java

```diff
@@ -4,14 +4,15 @@
 import htsjdk.samtools.cram.ref.ReferenceContext;
 import htsjdk.samtools.cram.structure.Container;
 import htsjdk.samtools.cram.structure.CramCompressionRecord;
-import htsjdk.samtools.cram.structure.CramCompressionRecordUtil;
+import htsjdk.samtools.cram.structure.CRAMStructureTestUtil;
 import htsjdk.samtools.cram.structure.Slice;
-import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.util.List;
 
+import static htsjdk.samtools.cram.structure.CRAMStructureTestUtil.READ_LENGTH_FOR_TEST_RECORDS;
```

Comment:
Folklorish: Even though they're all over the place in htsjdk, we generally try to avoid use of static imports in new code.

Reply:
OK - will revert